intersphinx_registry/lookup.py

```diff
@@ -71,13 +92,32 @@ def main():
             filtered.append((key, entry, proj, version, display_name, url_path))
             width = [max(w, len(x)) for w, x in zip(width, item)]
 
-
     for key, entry, proj, version, display_name, url_path in filtered:
         w_key, w_entry, w_proj, w_version, w_di, w_url = width
         print(
-            f"{key:<{w_key}}  {entry:<{w_entry}}  {proj:<{w_proj}}  {version:<{w_version}}  {display_name!r:<{w_di+2}}  {url_path}"
+            f"{key:<{w_key}}  {entry:<{w_entry}}  {proj:<{w_proj}}  "
+            f"{version:<{w_version}}  {display_name!r:<{w_di + 2}}  {url_path}"
         )
 
 
 if __name__ == "__main__":
-    main()
+    if len(sys.argv) not in [2, 3]:
+        sys.exit(
+            """Usage: python -m intersphinx_registry.lookup <package>[,package] [search_term]
+
+        Example:
+
+        $ python -m intersphinx_registry.lookup numpy,scipy array
+        $ python -m intersphinx_registry.lookup ipython formatters.html
+
+        """
+        )
+
+    packages_str = sys.argv[1]
+    search_term = sys.argv[2] if len(sys.argv) == 3 else None
+
+    try:
+        lookup_packages(packages_str, search_term)
+    except Exception as e:
+        print(f"Error: {e}", file=sys.stderr)
+        sys.exit(1)
```

Comment:
Did you know you can [`sys.exit(str)`](https://docs.python.org/3/library/sys.html#sys.exit)  instead of this pattern ?